File: New folder/notes.cs
// csharp notes
using System;
using System.Linq;

class Notes {
    static string Greet(string name) => $"hello {name}";

    static void Main() {
        Console.WriteLine(Greet("practice"));

        int[] nums = {1, 2, 3};
        var doubled = nums.Select(n => n * 2);
        Console.WriteLine(string.Join(",", doubled));
    }
}
// TODO: try LINQ deeper next

File: New folder/notes.go
// go notes
package main

import "fmt"

func greet(name string) string {
	return "hello " + name
}

func main() {
	fmt.Println(greet("practice"))

	nums := []int{1, 2, 3}
	for _, n := range nums {
		fmt.Println(n * 2)
	}
}

// TODO: try goroutines next
